test/rpc_test.py

```diff
@@ -90,16 +89,21 @@ def _wrap_with_rpc(test_method):
 
     @functools.wraps(test_method)
     def wrapper(self, *arg, **kwargs):
-        store = dist.FileStore(self.file_name, self.world_size)
-        dist.init_process_group(
-            backend="gloo", rank=self.rank, world_size=self.world_size, store=store
+        init_method = (
+            "file://{file_name}?rank={rank}&world_size={world_size}"
+        ).format(
+            file_name=self.file_name, rank=self.rank, world_size=self.world_size
         )
+
+        dist.init_process_group(backend="gloo", init_method=init_method)
```

Comment:
nit, should we use the `Backend` enum that is available in `distributed_c10d.py` and have `Backend.GLOO` here?

Reply:
@rohan-varma Good suggestion. Will do in later PRs.